Helper/StoreConfig.php

```diff
@@ -206,6 +212,13 @@ public function __construct(
         parent::__construct($context);
     }
 
+    public function createStore(array $storeData): array 
+    {
+        $managementClient = $this->managementClientFactory->create(['apiType' => 'doomanager']);
+        
+        return $managementClient->createStore($storeData);
```

Comment:
```suggestion
        return $this->managementClientFactory->create(['apiType' => 'doomanager']);
```

Reply:
This is not the same thing you had before.